client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-content/style.scss

```diff
@@ -1,3 +1,46 @@
 .site-content__mobile-view {
 	margin-bottom: 1.5rem;
 }
+
+.edit-all__bulk-actions {
+	margin: 10px;
+	padding-right: 5px;
```

Comment:
Could we use RTL-aware style attributes everywhere on this file? 

RTL equivalent properties

padding-right: padding-inline-end
padding-left: padding-inline-start
padding-top: padding-block-start
padding-bottom: padding-block-end

The same goes for margin.